airflow/dags/rt_loader/parse_rt_vehicle_positions.py

```diff
@@ -0,0 +1,143 @@
+# ---
+# python_callable: main
+# provide_context: true
+# ---
+
+from google.transit import gtfs_realtime_pb2
+from google.protobuf import json_format
+from google.protobuf.message import DecodeError
+from calitp.storage import get_fs
+import pandas as pd
+import tempfile
+import time
+
+
+def parse_pb(path, open_with=None):
+    """
+    Convert pb file to Python dictionary
+    """
+    open_func = open_with if open_with is not None else open
+    feed = gtfs_realtime_pb2.FeedMessage()
+    try:
+        feed.ParseFromString(open_func(path, "rb").read())
+        d = json_format.MessageToDict(feed)
+    except DecodeError:
```

Comment:
Why is this very lenient except block here? Did you encounter files that didn't parse?

Reply:
I encountered files that didn't parse.